### PR TITLE
DR-1850: Replace checkstyle with spotless

### PR DESCRIPTION
**Pull request by pshapiro4broad**
- update build to use spotless, remove checkstyle
- reformat code using google code conventions

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -30,15 +30,14 @@ plugins {
     id 'org.liquibase.gradle' version '2.0.1'
     id "org.gradle.test-retry" version "1.2.0"
     id 'antlr'
-    id 'com.github.spotbugs' version '4.7.0'
+    id 'com.github.spotbugs' version '4.7.1'
```
Comment (fboulnois) [inline build.gradle:33]: For others taking a look at this PR, the `build.gradle` files are the only two non-java files that are changed. All the other changes are due to spotless suggestions. The bulk of the changes are whitespace and documentation formatting.

**Review by fboulnois — APPROVED** [review]
lgtm assuming no changes in external behavior!

**Comment by fboulnois** [comment]
I guess you'll need to rebase off develop, too.

**Review by snf2ye — APPROVED** [review]
I didn't make it through all 500, but build.gradle and the first 100 look good to me!

**Comment by pshapiro4broad** [comment]
Merging up with errors; integration tests are failing due to infrastructure issues.

**Pull request merged**